packages/melange-radix-icons/melange-radix-icons.0.0.1/opam

```diff
@@ -0,0 +1,42 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Melange bindings for radix-icons"
+maintainer: ["I Putu Kevin Anantha Vijaya <kevin.anantha@gmail.com>"]
+authors: ["I Putu Kevin Anantha Vijaya <kevin.anantha@gmail.com>"]
+license: "MIT"
+homepage: "https://github.com/kevanantha/melange-radix-icons"
+bug-reports: "https://github.com/kevanantha/melange-radix-icons"
+depends: [
+  "dune" {>= "3.12" & >= "3.12"}
```

Comment:
```suggestion
  "dune" {>= "3.12"}
```
To fix this you just need to remove dune from the dependencies in the dune-project file (it is added automatically)